.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
What does `_` mean

Reply:
husky will create a new dir named `_`, it should be ignored.
![image](https://github.com/Magickbase/ckb-explorer-frontend/assets/20639676/e7cdb0bf-f5a3-4b7b-aab0-83417ee566f9)

Reply:
It could be annotated

Reply:
BTW, the whole directory is ignored by `_/.gitignore`

Reply:
https://github.com/webpack/webpack/tree/main/.husky I retrieve the Webpack repository, `.gitignore` is not necessary. Maybe the setup script of husky and lint-staged is outdated.